setup.cfg

```diff
@@ -15,8 +15,26 @@ python = python
 universal = 0
 
 [flake8]
-builtins = _
+filename =
+    *.py
+    ./linkchecker
```

Comment:
What does this do, exactly?

Reply:
That's for if you just type `flake8` in the top-level directory, else linkchecker is not scanned because it doesn't end in `.py` - the `./` does seem to matter.